src/sentry/notifications/utils/participants.py

```diff
@@ -289,8 +289,18 @@ def determine_eligible_recipients(
         if owners:
             return owners
 
-        if fallthrough_choice:
-            return get_fallthrough_recipients(project, fallthrough_choice)
+        if (
+            features.has(
+                "organizations:issue-alert-fallback-targeting",
+                project.organization,
+                actor=None,
+            )
+            and not fallthrough_choice
+        ):
+            logger.warning(f"Missing fallthrough type in project: {project}")
```

Comment:
I _think_ we should be able to guarantee a fallthrough choice here because of the default.